udp/src/main/scala/akka/stream/alpakka/udp/impl/UdpBind.scala

```diff
@@ -31,7 +34,10 @@ import scala.concurrent.{Future, Promise}
 
   override def preStart(): Unit = {
     implicit val sender = getStageActor(processIncoming).ref
-    IO(Udp) ! Udp.Bind(sender, localAddress)
+    if (options.nonEmpty)
+      IO(Udp) ! Udp.Bind(sender, localAddress, options)
+    else
+      IO(Udp) ! Udp.Bind(sender, localAddress)
```

Comment:
You can remove the check, not passing `options` is equal to passing an empty traversable:
https://github.com/akka/akka/blob/d83cda8f56aa97f4cd1d1d8e6c494fc2996818fa/akka-actor/src/main/scala/akka/io/Udp.scala#L104-L107